server/core/config/raw/repo_cfg_test.go

```diff
@@ -10,7 +10,7 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-func TestConfig_UnmarshalYAML(t *testing.T) {
+func TestConfig_UnmarshalYAML(t *testing.T) {parser_validator_test.go:1074:
```

Comment:
Is this right?

Reply:
Its still a wip, and not likely to be right yet :) will undraft it once i feel its ready for another round of reviews. :) a week of christmas vacation in Norway now so have high hopes of finding the time.

Reply:
Sure no worries. Take your time!